components/ReferenceManager/form/PrimaryButton.tsx

```diff
@@ -0,0 +1,37 @@
+import { ReactElement } from "react";
+import { Container, Box, Button } from "@mui/material";
```

Comment:
The recommended way to import is using tree shaking for reduced bundle size:

```
import Container from "@mui/material/Container";
import Box from "@mui/material/Box";
```

....

Reply:
I feel like webpack does automatic tree shaking so it shouldn't matter, but I might be wrong here.